src/derivative_operators/upwind_operator.jl

```diff
@@ -235,3 +235,43 @@ function LinearAlgebra.opnorm(A::UpwindOperator{T,S,LBC,RBC}, p::Real=2) where {
         opnorm(convert(Array,A), p)
     end
 end
+
+function LinearAlgebra.Array(A::UpwindOperator{T}) where T
```

Comment:
These upwind operators need to flip the stencils depending on the current direction. We can ignore this for now since they may change soon.

Reply:
@ChrisRackauckas it is not sufficient to check A.directions[i] in order to determine the direction of the ith point of the discretization?

Reply:
@ChrisRackauckas it is not sufficient to check A.directions[][i] for which stencil to use for the ith gird point?